kannon/master.py

```diff
@@ -49,16 +49,24 @@ def build(self, root_task: gokart.TaskOnKart) -> None:
         task_queue = self._create_task_queue(root_task)
 
         # consume task queue
-        launched_task_ids: Set[str] = set()
```

Comment:
`launched_task_ids` set is used to check if a given task is already assigned to k8s jobs, but it can be replaced with `self.task_ids_to_job_name` dict. So, it is removed.